includes/class-sensei-course.php

```diff
@@ -1209,16 +1172,13 @@ public function get_product_courses( $product_id = 0 ) {
 	 */
 	public static function get_product_courses_query_args( $product_id ) {
```

Comment:
This should have a `@deprecated` tag.

Reply:
https://github.com/Automattic/sensei/pull/2495